packages/docs/src/routes/docs/advanced/prefetching/index.mdx

```diff
@@ -46,8 +46,8 @@ export default function (opts: RenderToStreamOptions) {
 
 | Option              | Description                                                                                                                                                                                                                                                                                                 |
 | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `prefetchEvent`     | Dispatch a `qprefetch` event with detail data containing the urls that should be prefetched. The event dispatch script will be inlined into the document's HTML.                                                                                                                                            |
-| `linkInsert`        | Insert the `<link>` element into the document. When using `html-append`, it will render each `<link>` directly within html, appended at the end of the body. Using the `js-append` option will instead insert JavaScript, which will create the elements at runtime and append them at the end of the body. |
+| `prefetchEvent`     | Dispatch a `qprefetch` event with detailed data containing the urls that should be prefetched. The event dispatch script will be inlined into the document's HTML.                                                                                                                                            |
```

Comment:
It's references custom event's `detail` data. So maybe we put `detail` in code blocks instead.